### PR TITLE
Configuring an IDE: Add intro and details for scons path with Qt Creator

### PR DESCRIPTION
**Pull request by akien-mga**
Also moved Qt Creator up as it's the IDE used by most core developers.
Some users saw Kdevelop first and expected it to be the recommended one.

### CODE REVIEW

File: development/cpp/configuring_an_ide.rst [hunk 1]
```diff
@@ -140,6 +92,67 @@ clicking the "Apply Filter" button. A simple click on "Ok" and you're ready to c
 
 .. image:: img/qtcreator-edit-files-dialog.png
 
+Kdevelop
+--------
+
+It is a free, open source IDE for Linux, Solaris, FreeBSD, Mac OS X and other Unix flavors.
```
Comment (OvermindDL1) [inline development/cpp/configuring_an_ide.rst]: KDevelop also supports Windows, this should be updated:  https://www.kdevelop.org/download

Comment (akien-mga) [reply development/cpp/configuring_an_ide.rst]: Thanks, updated.

**Comment by OvermindDL1** [comment]
As another aside, KDeveop uses CMake as it's base project format (and has for a long while now), isn't there any way of having scons produce a CMakeLists.txt file so the user doesn't have to go through that pain of setting up a custom build system in it, because that is quite a pain?

**Comment by mhilbrunner** [comment]
@OvermindDL1 Couldn't find something like that, this is the only thing Google pointed me to: http://scons-dev.scons.narkive.com/fmxmIl1O/generate-cmakelists-with-scons

**Comment by OvermindDL1** [comment]
> Couldn't find something like that

Hmm, how sad...  Eh, if only godot used cmake, especially as how ubiquitous cmake has become (even boost is switching to it, which I find odd considering how hard up they were on b2...).

**Pull request merged**